frontend/src/lib/search.ts

```diff
@@ -46,10 +46,10 @@ export function queryStatusCount(cluster: string): IRequestResult<ISearchResult>
                 },
                 {
                     filters: [
-                        { property: 'compliant', values: ['!Compliant'] },
+                        { property: 'compliant', values: ['NonCompliant'] },
                         { property: 'kind', values: ['policy'] },
                         { property: 'namespace', values: [cluster] },
-                        { property: 'cluster', values: 'local-cluster' },
+                        { property: 'cluster', values: ['local-cluster'] },
```

Comment:
I think this might partially address https://issues.redhat.com/browse/ACM-2004
Still getting an error though:
{"errors":[{"message":"internal system error","path":["searchResult",1,"count"]},{"message":"internal system error","path":["searchResult",1,"related"]}],"data":{"searchResult":[{"count":5,"related":[{"kind":"Application","count":1,"__typename":"SearchRelatedResult"}],"__typename":"SearchResult"},{"count":null,"related":null,"__typename":"SearchResult"}]}}

Reply:
My aim was to address bug ACM-2004 (I didn't know a bug had been created already). I wasn't seeing the `internal system error` on my end but will try to recreate this.